tooling/bundler/src/bundle/windows/nsis.rs

```diff
@@ -341,6 +341,21 @@ fn build_nsis_app_installer(
   }
 
   let mut handlebars = Handlebars::new();
+  handlebars.register_escape_fn(|s| {
+    let mut output = String::new();
+    for c in s.chars() {
+      match c {
+        '\"' => output.push_str("$\\\""),
+        '$' => output.push_str("$$"),
+        '`' => output.push_str("$\\`"),
+        '\n' => output.push_str("$\n"),
+        '\t' => output.push_str("$\t"),
+        '\r' => output.push_str("$\r"),
```

Comment:
```suggestion
        '\n' => output.push_str("$\\n"),
        '\t' => output.push_str("$\\t"),
        '\r' => output.push_str("$\\r"),
```
we need it to render `$\t` not `$	`